src/current/_includes/v23.2/sql/grant-revoke-schema-changes.md

```diff
@@ -1,9 +1,19 @@
-`GRANT` and `REVOKE` operations are [schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}).  As such they inherit the [limitations of schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}#limitations).
+User/role management operations (such as [`GRANT`]({% link {{ page.version.version }}/grant.md %}) and [`REVOKE`]({% link {{ page.version.version }}/revoke.md %})) are [schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}). As such they inherit the [limitations of schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}#limitations).
 
-For example, schema changes wait for concurrent transactions using the same resources as the schema changes to complete. In the case of [role memberships]({% link {{ page.version.version }}/security-reference/authorization.md %}#roles) being modified by `GRANT` and `REVOKE` inside a transaction, most transactions need access to the set of role memberships.
+For example, schema changes wait for concurrent [transactions]({% link {{ page.version.version }}/transactions.md %}) using the same resources as the schema changes to complete. In the case of [role memberships]({% link {{ page.version.version }}/security-reference/authorization.md %}#roles) being modified inside a transaction, most transactions need access to the set of role memberships. Using the default settings, any modifications to roles require schema leases to expire, which can take up to 5 minutes.
```

Comment:
Unfortunately the schema lease is a different kind of lease (terminology overload) that still needs documentation. It's related to how the database does name resolution and the caching of names of things

We have an issue to document this ([DOC-877](https://cockroachlabs.atlassian.net/browse/DOC-877)) which says in part that

> what we need is to explain to users that, in the query path, we need to resolve the names, but that the names are cached using leases that expire every 5 minutes. it’s pretty important to have this part documented, because it does come up from time to time when people run into problems if a node dies unexpectedly that is holding a lease - that can lead to some unexpected waits when doing a schema change on that object that was leased

and if you look at that issue you see a convo where it's explained "no this is a different lease" which .. is really unfortunate.  But hopefully I can work on DOC-877 Soon (tm).